DAG unittest: `TestBBoltDAG_Diagnostics` fails on arm64

The test failed on my Apple Macbook M1 (and in a clean docker environment using the `arm64` platform).
It succeeds when switching to `amd64` in docker.

Logs:

```
--- FAIL: TestBBoltDAG_Diagnostics (0.11s)
    bbolt_dag_test.go:238: 
                Error Trace:    bbolt_dag_test.go:238
                Error:          Not equal: 
                                expected: "[DAG] Heads: [3a3b3e2ada3df88bcede97f7ff4a05c29ed2c2a9c282fd0f599fbd1cb732d0b0]\n[DAG] Number of transactions: 1\n[DAG] Stored transaction size (bytes): 8192"
                                actual  : "[DAG] Heads: [3a3b3e2ada3df88bcede97f7ff4a05c29ed2c2a9c282fd0f599fbd1cb732d0b0]\n[DAG] Number of transactions: 1\n[DAG] Stored transaction size (bytes): 32768"
                                
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -2,2 +2,2 @@
                                 [DAG] Number of transactions: 1
                                -[DAG] Stored transaction size (bytes): 8192
                                +[DAG] Stored transaction size (bytes): 32768
                Test:           TestBBoltDAG_Diagnostics
time="2021-07-05T11:04:35+02:00" level=error msg="Unable to read payload to publish DAG: (ref=aea340244cc662167ac6ed4b192512a2fc29026eb0f6e3b00ec13db7868257bd) failed" module=Network
time="2021-07-05T11:04:35+02:00" level=error msg="Transaction subscriber returned an error (ref=744d4d985b4129b0bd7ee64ae89f80ecea79f5459e73b27dbfef7ea16ebf158d,type=foo/bar): failed" module=Network
FAIL
FAIL    github.com/nuts-foundation/nuts-node/network/dag        4.439s
```

--- a/network/dag/bbolt_dag.go
+++ b/network/dag/bbolt_dag.go
@@ -96,7 +96,7 @@ type dataSizeStatistic struct {
 }
 
 func (d dataSizeStatistic) Name() string {
-	return "[DAG] Stored transaction size (bytes)"
+	return "[DAG] Stored database size (bytes)"
 }
 
 func (d dataSizeStatistic) String() string {

--- a/network/dag/bbolt_dag_test.go
+++ b/network/dag/bbolt_dag_test.go
@@ -20,6 +20,7 @@ package dag
 
 import (
 	"errors"
+	"fmt"
 	"sort"
 	"strings"
 	"testing"
@@ -234,10 +235,15 @@ func TestBBoltDAG_Diagnostics(t *testing.T) {
 		lines = append(lines, diagnostic.Name()+": "+diagnostic.String())
 	}
 	sort.Strings(lines)
+
+	dbSize := dag.Statistics()
+	assert.NotZero(t, dbSize)
+
 	actual := strings.Join(lines, "\n")
-	assert.Equal(t, `[DAG] Heads: [`+doc1.Ref().String()+`]
+	expected := fmt.Sprintf(`[DAG] Heads: [`+doc1.Ref().String()+`]
 [DAG] Number of transactions: 1
-[DAG] Stored transaction size (bytes): 8192`, actual)
+[DAG] Stored database size (bytes): %d`, dbSize.DataSize)
+	assert.Equal(t, expected, actual)
 }
 
 func Test_parseHashList(t *testing.T) {

